public/src/main/java/com/revenuecat/purchases/models/StoreTransaction.kt

```diff
@@ -87,10 +88,12 @@ data class StoreTransaction(
     /**
      * Skus associated with the transaction
      */
+    @IgnoredOnParcel
     @Deprecated(
         "Replaced with productIds",
         ReplaceWith("productIds")
     )
+    // TODO should we access skus in originaljson
```

Comment:
Hmm I have no idea so would there be any difference between accessing `productIds` directly and this? Ideally the information would be the same and `productIds` is easier to access